conf.py

```diff
@@ -47,7 +47,7 @@
 
 # General information about the project.
 project = u'Syncthing'
-copyright = u'2015, The Syncthing Authors'
+copyright = u'2017, The Syncthing Authors'
```

Comment:
No! No No No!

The date is the date at which you claim copyright on the work. If you update this to 2017, you're claiming that you wrote the work in 2017. If someone copied the work in 2016, but you're claiming you wrote it in 2017, your claim to copyright looks very shaky.

[More info on SO](http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated/2391555#2391555).